docs/user-guide.md

```diff
@@ -44,4 +46,44 @@ Flags:
       --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging
 
 Use "descheduler [command] --help" for more information about a command.
-```
\ No newline at end of file
+```
+
+## Production Use Cases
+This section contains descriptions of real world production use cases.
+
+### Balance Cluster By Pod Age
+When initially migrating applications from a static virtual machine infrastructure to a cloud native k8s
+infrastructure there can be a tendency to treat application pods like static virtual machines. One approach
+to helping developers and operators break this bad habit is to ensure that pods only run for a fixed amount
```

Comment:
nit: @seanmalloy could you rephrase this line a bit specifically avoiding "bad habit"?

Reply:
Fixed. The phrase "bad habit" has been removed.